libs/coin-modules/coin-cosmos/src/api/Cosmos.ts

```diff
@@ -445,7 +450,9 @@ export class CosmosAPI {
    * @deprecated body {..., mode } -> BROADCAST_MODE_BLOCK (Deprecated: post v0.47 use BROADCAST_MODE_SYNC instead)
    * @notice returns {..., events } (Since: cosmos-sdk 0.42.11, 0.44.5, 0.45)
    */
-  broadcast = async ({ signedOperation: { operation, signature } }): Promise<Operation> => {
+  // TO CHECK
+  broadcast = async (tx: any): Promise<Operation> => {
```

Comment:
We need to type tx if possible

Reply:
Done here, https://github.com/LedgerHQ/ledger-live/pull/7636/commits/a8afab5086b1b939e219325e38ad781c4e0a18e0